### ISSUE TITLE
"Warning: insufficient funds" when trying to send USDt

### ISSUE THREAD

**Issue by cbeams** [Blockstream/aqua_ios#5]
I'm running Aqua 1.0.4, freshly installed for the first time today.

I've funded it with a small amount of USDt:

![signal-2021-02-10-173738_002](https://user-images.githubusercontent.com/301810/107540808-c489e480-6bc6-11eb-95e4-1ca6ac5fb7c2.png)

But upon attempting to send a portion of it to another wallet:

![signal-2021-02-10-173738_004](https://user-images.githubusercontent.com/301810/107541067-09ae1680-6bc7-11eb-99a9-b9d98f1e4043.png)

I get the following error:

![signal-2021-02-10-173738_003](https://user-images.githubusercontent.com/301810/107541304-4843d100-6bc7-11eb-9caa-d79bb11b2c2a.png)

I have tried using different addresses from different wallet as well as different amounts to send, and the same error appears in every case. What could be the problem?

**Comment by LeoComandini** [comment]
Hi @cbeams,

Does your wallet have some L-BTC? Those are needed to pay the fees in the Liquid Network.

**Comment by cbeams** [comment]
No, it's my first time using Liquid and USDt. This makes sense when I hear it, and wondered if it might be a transaction fee problem, but wasn't obvious to me that the solution would be to fund the L-BTC wallet. Thanks in any case for the quick feedback.

**Comment by cbeams** [comment]
Funding the L-BTC wallet and re-trying the transaction from within the USDT wallet side worked as expected. Thanks again.
